src/Session.php

```diff
@@ -80,7 +81,7 @@ class Session extends \yii\web\Session
     public function init()
     {
         if (is_string($this->redis)) {
-            $this->redis = Yii::$app->get($this->redis);
+            $this->redis = Instance::ensure($this->redis, Connection::className());
```

Comment:
We could replace entire initialization of `$redis` property by `Instance::ensure()`.

https://github.com/yiisoft/yii2-redis/blob/3fe93b9208f97d16c0479511951db530a040de40/src/Session.php#L83-L93